carma-messenger-core/Dockerfile

```diff
@@ -39,9 +39,7 @@ LABEL org.label-schema.vcs-url="https://github.com/usdot-fhwa-stol/carma-messeng
 LABEL org.label-schema.vcs-ref=${VCS_REF}
 LABEL org.label-schema.build-date=${BUILD_DATE}
 
-COPY --from=setup /opt/carma/install /opt/carma/install
 COPY --from=setup /opt/carma/install_ros2 /opt/carma/install_ros2
-RUN sudo chmod -R +x /opt/carma/install
 
 RUN pip install future
 
```

Comment:
`roslaunch` command should also change to ROS2 version with appropriate launch python files.

Reply:
1. That reminds me, ROS1 style `launch` files should be removed. There is one functionality in the `carma-messenger.launch` file that should be converted to ROS2 as well. Not sure if we can do it similarly in ROS2, I assume we can.

```
  <!-- System Alert Publisher to publish simulated DRIVERS_READY status -->
  <node pkg="rostopic" type="rostopic" name="drivers_ready_rostopic_pub"
        args="pub -l /system_alert cav_msgs/SystemAlert '{ type: 5, description: Simulated Drivers Ready }'"
  />
``` 

2. all the `docker-compose.yml` types in `carma-messenger-config` should drop the roscore and ros1_bridge I think because no ROS1 component is running. 
3. these changes require integration testing. 

However, I am also okay holding off on the changes suggested in this particular comment, if your current change is enough to unblock you towards your C1T efforts.

Reply:
Are we planning to migrate bagging to the ROS2 launch file as well?

Reply:
Yes! We do need the ros2 bag as well although no specific work is setup, so it should done as part of this effort.
Example where we call the ros2 bag here carma-platform https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/carma/launch/carma_src.launch.py#L196